test/core/transport/binder/end2end/echo.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A simple RPC service that echos what the client passes in. The request and
+// the response simply contains the text represented in a string.
+//
+// This service is for end-to-end testing with fake binder tunnels.
+syntax = "proto3";
+
+package grpc_binder.end2end_testing;
```

Comment:
Is this package name consistent with the rest of proto in the repo?

Reply:
I've seen different kinds of package names, most of them are either just the filenames of the proto or `grpc.testing`. Not sure about whether there's a rule behind it though.

BTW, there's already an `echo.proto` in the repo that offers more functionalities than this one does. Maybe we could just use the existing one and remove `echo.proto` from this PR?

Reply:
Got it. Yeah I think we can add a TODO and do it later. (Because we want to avoid touching existing code in this PR)

Reply:
Sure, I can do that in upcoming PRs.